internals/daemon/daemon.go

```diff
@@ -776,6 +776,12 @@ func (d *Daemon) Dying() <-chan struct{} {
 	return d.tomb.Dying()
 }
 
+// Err returns the death reason, or ErrStillAlive
+// if the tomb is not in a dying or dead state.
```

Comment:
Comment taken from [`d.tomb.Err()`](https://pkg.go.dev/gopkg.in/tomb.v2#Tomb.Err)'s documentation.